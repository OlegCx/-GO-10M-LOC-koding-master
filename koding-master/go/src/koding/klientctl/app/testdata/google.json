{
  "provider": {
    "google": {
      "credentials": "${var.google_credentials}",
      "project": "${var.google_project}",
      "region": "${var.google_region}"
    }
  },
  "resource": {
    "google_compute_instance": {
      "google-instance": {
        "disk": {
          "image": "ubuntu-1404-lts"
        },
        "machine_type": "f1-micro",
        "metadata": {
          "user-data": "\necho \"hello world!\" >> /helloworld.txt\n"
        },
        "name": "koding-${var.koding_group_slug}-${var.koding_stack_id}-${count.index+1}",
        "zone": "us-central1-a"
      }
    }
  }
}
