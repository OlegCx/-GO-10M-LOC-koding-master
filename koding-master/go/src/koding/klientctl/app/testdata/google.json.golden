{
  "provider": {
    "google": {
      "credentials": "${var.google_credentials}",
      "project": "${var.google_project}",
      "region": "${var.google_region}"
    }
  },
  "resource": {
    "google_compute_instance": {
      "google-instance": {
        "disk": {
          "image": "ubuntu-1404-lts"
        },
        "koding_always_on": true,
        "machine_type": "f1-micro",
        "metadata": {
          "user-data": "#cloud-config\nruncmd:\n- echo \"hello world!\" >> /helloworld.txt\nssh_authorized_keys:\n- ssh-rsa AAAAB3NzaC1yc2EA... koding-350298856\n"
        },
        "name": "koding-${var.koding_group_slug}-${var.koding_stack_id}-${count.index+1}",
        "zone": "us-central1-a"
      }
    }
  }
}
